README.md

```diff
@@ -8,3 +8,165 @@ You can choose to use either the:
 - [React COGS SDK](./packages/javascript/README.md)
 
 Learn more in the [COGS technical documentation](https://docs.cogs.show).
+
+## Quick start
+
+Add the COGS JavaScript SDK to your project:
+
+```bash
+npm install --save @clockworkdog/cogs-client
+# OR
+yarn add @clockworkdog/cogs-client
+```
+
+or if you are using React:
+
+```bash
+npm install --save @clockworkdog/cogs-client-react
+# OR
+yarn add @clockworkdog/cogs-client-react
+```
+
+## COGS Media Master custom content
+
+### What is a Media Master custom content?
+
+COGS Media Masters can display a browser window with custom HTML, Javascript and CSS, using the COGS SDK.
+
+Some other features you can add to your custom content include:
+
+- Play audio from your COGS project
+- Display or react to COGS text hints
+- Show the COGS show timer
+- Connect to MIDI or WebSerial devices
+
+### Contents of Media Master custom content
```

Comment:
This title is a little confusing with "content" used in different contexts.

How about "Structure of..."?

```suggestion
### Structure of Media Master custom content
```